framework/test/mocha/unit/modules/chain/chain.js

```diff
@@ -438,29 +438,6 @@ describe('Chain', () => {
 		});
 	});
 
-	describe('#_startLoader', () => {
-		beforeEach(async () => {
-			await chain.bootstrap();
-			sinonSandbox.stub(chain.loader, 'loadUnconfirmedTransactions');
-		});
-
-		it('should return if syncing.active in config is set to false', async () => {
-			// Arrange
-			chain.options.syncing.active = false;
-
-			// Act
-			chain._startLoader();
-
-			// Assert
-			expect(stubs.jobsQueue.register).to.not.be.called;
-		});
-
-		it('should load transactions and signatures', async () => {
```

Comment:
Maybe this test needs to be kept and make sure `this.loader.loadUnconfirmedTransactions` is being called?

Reply:
👍